### PR TITLE
Add conversation summary

### PR DESCRIPTION
**Pull request by nounder**
## Description of the Change

https://github.com/neynarxyz/monorepo/pull/1116

### New Endpoints

- `GET /farcaster/cast/conversation/summary`

## Checklist

- [x] I have validated that all new/updated endpoints conform to OAS standards.
- [x] I have updated or added necessary schema definitions.
- [x] I have ensured that the new schema I have added doesn't exist.
- [x] I have added description wherever necessary.
- [x] I have ensured that endpoint's responses are correct.
- [x] I have considered backward compatibility with previous versions of the API.
- [x] I have communicated any breaking changes to the appropriate stakeholders.
- [x] I have tested the OAS changes using a tool like [Swagger editor](https://editor.swagger.io/)

### CODE REVIEW

**Comment by nounder** [comment]
@manan19 on the checklist there is "I have tested the OAS changes using a tool like [Swagger editor](https://editor.swagger.io/)"

When I copy-paste spec to the editor it fails (even on main branch) with the following. Is it a leftover from past? Do we need this checklist item?

CleanShot 2024-12-17 at 22 01 05@2x

**Comment by Shreyaschorge** [comment]
> @manan19 on the checklist there is "I have tested the OAS changes using a tool like [Swagger editor](https://editor.swagger.io/)"
> 
> When I copy-paste spec to the editor it fails (even on main branch) with the following. Is it a leftover from past? Do we need this checklist item?
> 
> CleanShot 2024-12-17 at 22 01 05@2x

@nounder 

This version of the swagger editor supports v3.0.0
we moved from OAS v3.0.0 to v3.1.0
[the new one is this.](https://editor-next.swagger.io/?_gl=1*ad75lu*_gcl_au*MjA3NjEyNzk1Mi4xNzMzNDM5MDcw)
Thanks for pointing this out, will update it

File: src/v2/spec.yaml [hunk 1]
```diff
@@ -2176,6 +2176,31 @@ components:
                 $ref: "#/components/schemas/CastWithInteractions"
         next:
           $ref: "#/components/schemas/NextCursor"
+    ConversationSummary:
+      type: object
+      required:
+        - summary
```
Comment (Shreyaschorge) [inline src/v2/spec.yaml:2182]: Should this be called `conversation` cuz there is no `summary` property in this object

**Review by Shreyaschorge — CHANGES_REQUESTED** [review]
@nounder 
Small change.
Rest looks good

**Comment by nounder** [comment]
@Shreyaschorge thanks for nothing a mistake. It was fixed and a version bumped

**Review by Shreyaschorge — APPROVED** [review]
LGTM

**Pull request merged**